### ISSUE TITLE
The training/validation curves on the ImageNet dataset

### ISSUE THREAD

**Issue by shermanlian** [insitro/ChannelViT#16]
Hi! 

Do you have the training/validation curves for ChannelViT on the ImageNet dataset? I am running your code on ImageNet and found that the accuracy is about 60% with epoch 60th. The training is still running but I just want to check if my command is correct (since the accuracy achieved 74% in your paper with 100 epochs). And do you have any tips to train the model on ImageNet?

Thank you.

**Comment by srinivasans-insitro** [comment]
Hi @shermanlian!

Here's the validation accuracy curve for ChannelViT on the ImageNet dataset. Validation accuracy of ~60% at epoch 60 is expected. 
![Image](https://github.com/user-attachments/assets/11db91d5-692c-4c43-b3cb-78524bbdaab8)

**Comment by shermanlian** [comment]
Awesome, thank you! We successfully reproduced the result from your code too. Thank you for sharing!
